docs/source/install.rst

```diff
@@ -7,13 +7,13 @@ Prerequisites
 
 * If using the rocBLAS backend on an AMD machine:
 
-  * A ROCm enabled platform, more information `ROCm Documentation <https://rocm.github.io/>`_.
+  * A ROCm enabled platform, more information `ROCm Documentation <https://docs.amd.com/>`_.
   * A compatible version of rocBLAS
   * A compatible version of rocSOLVER for full functionality
 
 * If using the cuBLAS backend on a Nvidia machine:
 
-  * A HIP enabled platform, more information `HIP installation <https://rocmdocs.amd.com/en/latest/Installation_Guide/HIP-Installation.html#nvidia-platform/>`_.
+  * A HIP enabled platform, more information `HIP installation <https://docs.amd.com/bundle/HIP_Installation/page/HIP_Installation.html/>`_.
```

Comment:
You should drop the trailing slash. Our docs hosting website seems to be very forgiving of this sort of mistake, but it's kinda weird.

The id attribute of the "NVIDIA platform" heading on the new page is `d2e131`, so you should be able to link directly to it with https://docs.amd.com/bundle/HIP_Installation/page/HIP_Installation.html#d2e131.